frameworks/digital-outcomes-and-specialists/questions/brief-responses/newAvailabilityOutcomes.yml

```diff
@@ -0,0 +1,19 @@
+name: Earliest start date
+id: availability
+question: When is the earliest the team can start
```

Comment:
You can't define a `slug` for a section. You can define a `slug` for a question but this doesn't get passed up to the section.